packages/graphql-codegen-cli/src/cli.ts

```diff
@@ -31,7 +31,7 @@ export async function runCli(cmd: string): Promise<number> {
 export async function ensureGraphQlPackage() {
   try {
     await import('graphql');
-  } catch (e) {
+  } catch {
```

Comment:
It's complaining we have `e` but it's never used. I think we could do `_e` but removing the variable like this works too